File: SistemaParadaCerta/APIs-ParadaCerta/paradacerta-api/src/main/java/com/paradacerta/api/model/ConfirmacaoReservaResponse.java
package com.paradacerta.api.model;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.math.BigDecimal;

/**
 * Resposta da confirmação de reserva via QR Code.
 * Retornada pelo POST /api/reserva/confirmar quando o motorista escaneia
 * com sucesso o QR de confirmação no estacionamento.
 */
@Data
@AllArgsConstructor
public class ConfirmacaoReservaResponse {
    private String sessaoId;
    private Integer estacionamentoId;
    private String estacionamentoNome;
    private String pixKey;
    private long dataHoraConfirmacao; // Unix timestamp em ms
    private BigDecimal precoHora;
}

File: SistemaParadaCerta/AplicacaoMobile-ParadaCerta/ParadaCerta/app/src/main/java/com/example/paradacerta/models/ConfirmacaoReservaResponse.kt
package com.example.paradacerta.models

/** Resposta de POST /api/reserva/confirmar */
data class ConfirmacaoReservaResponse(
    val sessaoId: String,
    val estacionamentoId: Int,
    val estacionamentoNome: String,
    val pixKey: String?,
    val dataHoraConfirmacao: Long,
    val precoHora: Double
)
